src/web/mjs/connectors/PiccomaFR.mjs

```diff
@@ -1,99 +1,5 @@
WRJdH8k4WOHv','WOVdMJZcGCkN','CbVcSW','D8oHW7JdUmkK','WRJdUxjkwq','deldS8khbmoOxuRdSu/dNSoFW7G','we8Comof','W6tcMmkLdqS','bSooC8k/EG','dZDuuCkPFmoGWRRcT8o0iG','W7WeW4FcLSo2','yaRdKw8V','WQBdMmkmWQtdQG','AtldTh3cTSkHamoi','WQhdTXJcTSknW6S0W7mhd2Pl','WOdcKaZcPYWpWQBdMa','A8kRWO4uCW','WRmGWRK/s153uhacW7ZcHHG','dSkKnSoe','WP94W4Hsnq','WRRcScpdPSkL','W6pcOmkuWO45','WPhcHSo9DCon','nxBdRmoLkmoWB8oc','wtNdLSodW7K','imoCiIii','W4OTW4pcKCoYWPb6sq','pxZdNG','sSkaW7hdPCkH','sqpcICouya','W5CGW7FdTfhcHINcMxO','gxtdQmoJgmo9','pvxdR8oadW','xqFdMCo5W5K','WRNdSNLZwG','zSoXcCoBb8kVmCkcW4tdVCkGxa','i8oXWP/dQCkl','nSobWQldVCkc','WPemWRfxAG','vSkYk8ooyCojWPfcgaJcLce','sgHXWR7cSSoHkIq5F8ouwW','WP3dOrJdPSkQ','h8oRadKt','DSoTb8ocfW','F8k4W6xdSSkq','eXVcSCoEwq','WOBcOmoPuCoTW7xcN8o3','o8kkfSoeWRq','FmoDfmocha','WQdcUbfNkCo8b8kTW6KhWOldMrldNSo8WQHOWRBcQsGAjN9RW7VdGSocCmkPW6ddP8kbASkscq','WRRdOvDZsG','WRBdKvrVFW','WQtdPbJdKCk6','W5hdImkkWQG6','W7tdVrLMASo6mSkLW68oWORdMa','WPFcOCom','wcJcUmkNsSkMoSoKW5KOWOKIaG','WOhcQSooDCoTW7pcHCo/W7xcLCoGfMLh','W4FdGmk9WRuK','WOtcHXtcLWS','W7hdR8kKWPi','W7BdHmkMWRqs','b3KLW6ZdTq','u8kIW6RdPCk0c0W5gSkO','W7NcP8koWQyu','BmkYWP0lEG','cdjJW5icWPjl','xI1XW6ySW5u','gSo9ibCZ','DG7cT3ZdSq','dCoYiWSz','BCoIW4xcK8kkoW','W7BdR8kJWPC','WPqnoCkpqG','WQJcVaddHmkZW73dSa','nmkgWRZdHSkVW41ZrmofWQGyWPfKmmompSkVWPdcT8oPWOtcG1VdTCk1WPFcJ2hcKmorW5ZdJ8o8WPPUWQH/nZ01kSoZW7n9WP80W54CW7JcVXFcV8kLtmknsmoAdCoVWR9lDCo+W60','pv3cSdtdLXj/fualhG','W7VdMvS6','q8oGW68','DJddL00r','xxibamoOFCoxWPFcNSoubCkHW63cIL81bCov','W7jckCkGW4JcKxjUW6frWQvlWQZdLSo8sq','uNFdOv1TBc7cJ8kV','fvldUNxcHa','W6ldMG5yw8kN','xmoBomoUfG','D0xdT39L','WQSzbCkwsG','oaNdQ0yWz8oK','W5WNW4C','gI1PW6G/','iCohWQ/dI8kAWP9jzSo6','x8kYWQKHuq','eMe3W7pdIq','W7VcT8khWPOS','wbVcG3hdPa','y0NcUK7dGW','WQddICk2W7D1','W5XOW6tdO1tcIWVcNxPSW6dcJG','ALf+W6pdOW','WO7cRsddGmkk','W7jFiCkXW5xcMa','FCkwWOGQDwTctYZdKSkOeW','WQVdMG3cNmkx','fmoaWQhdM8kO','iSoxbXFcLa','jrGICea','W4VdJHzDwG','WPlcTIRdOCkB','W7JdHSkKWQiS','WQBcPt7dL8kWWPPopgPDhW','lNXEWO/cL8oRdG','tY7cHwxdIG','dSoMsmkoEW','WO1sW4HIlG','WPxdG8klW719W57dISovyW7dSW','WQ5KW75znq','b2WgW4O','W7yhW67cNSoO','Bg/dO3v2','WOXUW5TxaW','WPhcKaJcOdSp','F8o2a8omaCkJpmo8W5BdRG','xf5YWR3cSG','WQdcIf8AdCkRbSotBGBcQq','WOmoomkjyG','WOpdQCkqW7T1','W4vfiCoWW4lcLxznW6fgWRikW6y','dZhdVCo5hSk1x8o0W6jRWQKyiNBdPdzntMj9WP4XW7a/oxddNfRcSConj1BcQSkHWR7cKmoRgSoBq8kjWPBdNKf+W5JdKCkomXxdJmknvY3cU8kCvL/dKSkTWOG4aSoq','bmoYjr3cNW','jhtdMNhcLG','dmkGm8ozWQz9W63dSSoiWRe/WRykW6i','smkcjWvg','W6VdUrPXzq','Dt7dMwya','pmkVvmkzxmo5BmoHW5JdKSkTDSkNWOe','W4pcKmkXWPWR','W6vemmk8W4q','aJK+xuW','BComW6FcJ8kC','W6BdIs95Ca','WQZcGGVcSG4CWQddHbi','x8oJW4ZdH8kmiw3dN8oEWOFcVCkPWRe','u2DWWRRcNW','a2iHW5BdS8oQmCkV','W6NcHWiOsSoixCkXDCkh','W5z7W60PgWaJBfWJW7ZdK1i','b8kknCojWPq','t3pdQ1DpzW','WRGipSkfu3xcP3tdQYrwW79PwqTEWPBdKK5P','cmoQWPbg','WQ54W7Ttja','tbxdQvir','B8oKW4hcGSkQ','DeNcP3xdGHm','WQpcHL4CcCo8qSotxIVcQcFdOG','FSo5W4u','uqVdJNxcVG','W6NcJCk2dae'];_0x3b07=function(){return _0x61e65b;};return _0x3b07();}
```

Comment:
Where is this code from?
I think it is risky to execute code that is not fully understand directly within the electron process.

Reply:
We shouls discuss it privately.

Reply:
would it be fine to include it instead?